css/syntaxes.json

```diff
@@ -677,6 +692,9 @@
   "side-or-corner": {
     "syntax": "[ left | right ] || [ top | bottom ]"
   },
+  "sin()": {
+    "syntax": "sin( [ <number> | <angle> ] )"
```

Comment:
Why `<number>` and not something like `<calc-sum>`?

This syntax makes `sin(2 * 0.5)` invalid.

Reply:
Because this is what the spec says https://www.w3.org/TR/css-values-4/#trig-funcs

Reply:
My mistake, found this https://www.w3.org/TR/css-values-4/#calc-syntax

You are correct. I'll fix the PR.